### PR TITLE
Fix grammar and make text more concise.

### PR DESCRIPTION
**Pull request by jeannekamikaze**
- Change `Configure Boot Options` to `Boot Options` so that it matches the menu option in the image above.
- Make text constructs more consistent across the document.
- Various [which vs that](https://owl.purdue.edu/owl/general_writing/grammar/that_vs_which.html) fixes.
- `may` vs `might`.
- `login`/`log in` noun vs verb.
- Remove quotes from a handful of cross-references for consistency with others.
- A few commas here and there.

### CODE REVIEW

File: documentation/content/en/books/handbook/bsdinstall/_index.adoc [hunk 1]
```diff
@@ -541,13 +541,13 @@ Configuring too little swap can lead to inefficiencies in the VM page scanning c
 
 On larger systems with multiple SCSI disks or multiple IDE disks operating on different controllers, it is recommended that swap be configured on each drive, up to four drives.
 The swap partitions should be approximately the same size.
-The kernel can handle arbitrary sizes but internal data structures scale to 4 times the largest swap partition.
+The kernel can handle arbitrary sizes, but internal data structures scale to 4 times the largest swap partition.
 Keeping the swap partitions near the same size will allow the kernel to optimally stripe swap space across disks.
 Large swap sizes are fine, even if swap is not used much.
 It might be easier to recover from a runaway program before being forced to reboot.
 
 By properly partitioning a system, fragmentation introduced in the smaller write heavy partitions will not bleed over into the mostly read partitions.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/bsdinstall/_index.adoc:549]: Might as well use "write-heavy" here while you're in the neighborhood.

File: documentation/content/en/books/handbook/bsdinstall/_index.adoc [hunk 2]
```diff
@@ -967,19 +966,19 @@ But their use is encouraged.
...
-* `random_pid` - Randomize the PID of newly created processes.
+* `hide_uids` - Hide processes running as other users (UID). This prevents unprivileged users from seeing running processes from other users.
+* `hide_gids` - Hide processes running as other groups (GID). This prevents unprivileged users from seeing running processes from other groups.
+* `hide_jail` - Hide processes running in jails. This prevents unprivileged users from seeing processes running inside jails.
+* `read_msgbuf` - Disable reading kernel message buffer for unprivileged users. Prevent unprivileged users from using man:dmesg[8] to view messages from the kernel's log buffer.
+* `proc_debug` - Disable process debugging facilities for unprivileged users. Disables a variety of unprivileged inter-process debugging services, including some procfs functionality, `ptrace()`, and `ktrace()`. Please note that this will also prevent debugging tools such as man:lldb[1], man:truss[1] and man:procstat[1], as well as some built-in debugging facilities in certain scripting language like PHP.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/bsdinstall/_index.adoc:975]: "scripting languages".

File: documentation/content/en/books/handbook/bsdinstall/_index.adoc [hunk 3]
```diff
@@ -984,7 +984,7 @@ Here is a summary of the options that can be enabled in this menu:
 === Add Users
 
 The next menu prompts to create at least one user account.
-It is recommended to login to the system using a user account rather than as `root`.
+It is recommended to log in to the system using a user account rather than as `root`.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/bsdinstall/_index.adoc:987]: I'm on the fence between "log in to" and "log into".

File: documentation/content/en/books/handbook/bsdinstall/_index.adoc [hunk 4]
```diff
@@ -1005,23 +1005,23 @@ Here is a summary of the information to input:
...
+* `Login group` - The user's group. This is typically left blank to accept the default.
 * `Invite _user_ into other groups?` - Additional groups to which the user will be added as a member. If the user needs administrative access, type `wheel` here.
 * `Login class` - Typically left blank for the default.
-* `Shell` - Type in one of the listed values to set the interactive shell for the user. Refer to crossref:basics[shells,“Shells”] for more information about shells.
+* `Shell` - Type in one of the listed values to set the interactive shell for the user. Refer to crossref:basics[shells,Shells] for more information about shells.
 * `Home directory` - The user's home directory. The default is usually correct.
 * `Home directory permissions` - Permissions on the user's home directory. The default is usually correct.
 * `Use password-based authentication?` - Typically `yes` so that the user is prompted to input their password at login.
-* `Use an empty password?` - Typically `no` as it is insecure to have a blank password.
+* `Use an empty password?` - Typically `no` for security.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/bsdinstall/_index.adoc:1016]: I think I'd leave the specific reason spelled out here, maybe reworded: "as empty or blank passwords are insecure"

File: documentation/content/en/books/handbook/bsdinstall/_index.adoc [hunk 5]
```diff
@@ -1068,23 +1068,23 @@ image::bsdinstall-mainexit.png[]
...
 
-The messages that appeared during boot can be reviewed by pressing kbd:[Scroll-Lock] to turn on the scroll-back buffer.
+The messages that appear during boot can be reviewed by pressing kbd:[Scroll-Lock] to turn on the scroll-back buffer.
 The kbd:[PgUp], kbd:[PgDn], and arrow keys can be used to scroll back through the messages.
 When finished, press kbd:[Scroll-Lock] again to unlock the display and return to the console.
 To review these messages once the system has been up for some time, type `less /var/run/dmesg.boot` from a command prompt.
 Press kbd:[q] to return to the command line after viewing.
 
-If sshd was enabled in <<bsdinstall-config-serv>>, the first boot may be a bit slower as the system will generate the RSA and DSA keys.
+If sshd was enabled in <<bsdinstall-config-serv>>, the first boot might be a bit slower as the system generates RSA and DSA keys.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/bsdinstall/_index.adoc:1085]: "RSA and DSA" is no longer true by default. From /etc/rd.d/sshd:
> : ${sshd_rsa_enable:="yes"}
> : ${sshd_dsa_enable:="no"}
> : ${sshd_ecdsa_enable:="yes"}
> : ${sshd_ed25519_enable:="yes"}

But that may be overriden, so maybe be vague and use "SSH host keys".

**Review by pauamma — CHANGES_REQUESTED** [review]

**Comment by pauamma** [comment]
Again, thanks for this. I'm naturally nitpicky but it looks good.

File: documentation/content/en/books/handbook/bsdinstall/_index.adoc (hunk 3 above)
Comment (jeannekamikaze) [reply documentation/content/en/books/handbook/bsdinstall/_index.adoc:987]: Yeah, you're right. https://www.grammarly.com/blog/into-vs-in-to/

File: documentation/content/en/books/handbook/bsdinstall/_index.adoc (hunk 1 above)
Comment (jeannekamikaze) [reply documentation/content/en/books/handbook/bsdinstall/_index.adoc]: Done, thank you.

File: documentation/content/en/books/handbook/bsdinstall/_index.adoc (hunk 4 above)
Comment (jeannekamikaze) [reply documentation/content/en/books/handbook/bsdinstall/_index.adoc]: Reworded, thank you.

**Review by pauamma — APPROVED** [review]

**Comment by dbaio** [comment]
Committed on 1c0e8be1cf95d42b6a30214146f907579cf87910. Thank you.
